go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	github.com/stretchr/testify v1.8.0
 	github.com/u-root/u-root v7.0.0+incompatible
 	github.com/urfave/cli/v2 v2.11.1
+	github.com/vishvananda/netlink v1.1.1-0.20211118161826-650dca95af54
```

Comment:
Can our [existing ghw library](https://github.com/jaypipes/ghw) do this already? If so, let's use that instead of bringing in a redundant library.

Reply:
`jaypipes/ghw` doesnt allow me to query relationship between interfaces and if they are involved in a bond interface.

netlink allows me to query the bo/br interfaces, identity their index and query all other interfaces which may have master index set to index of either bo/br. 

Once the physical interface names are available, `ghw` is used to query their pci addresses and subsequently skip the same from pcidevice creation logic.